config/environment.js

```diff
@@ -31,6 +31,7 @@ module.exports = function (environment) {
       authRedirectUrl: '{{ACMIDM_AUTH_REDIRECT_URL}}',
       switchRedirectUrl: '{{ACMIDM_SWITCH_REDIRECT_URL}}',
     },
+    adminRole: 'LoketAdmin',
```

Comment:
Since we can't know the real admin role yet I would revert back to the variable version and merge the PR. We can then adjust it in the backend once we actually know.